Dockerfile

```diff
@@ -37,11 +37,14 @@ FROM alpine:3.14
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
   copyright='Copyright (c) 2021: Intel'
 
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+RUN apk add --update --no-cache zeromq dumb-init
+
 COPY --from=builder /go/src/github.com/edgexfoundry/device-rfid-llrp-go/LICENSE /
```

Comment:
Set WORKDIR  above to /app
```
WORKDIR  /app
```
and replace `/go/src/github.com/edgexfoundry` with `/app` in this block

Reply:
Just want to be sure on this one:
`WORKDIR $GOPATH/src/github.com/edgexfoundry/device-rfid-llrp-go`   ->   `WORKDIR /app`

Reply:
Correct